packages/harpguru-core/src/components/option-title-stack/option-title-stack.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { OptionTitle } from '../option-title'
+import type { OptionStackProps } from '../option-stack'
+import type { WithTransition } from '../../types'
+import { useInterpolateOptionStackTransitionValue } from '../../hooks'
+
+export const OptionTitleStack = ({
+  optionPropsz,
+  transitionValue,
+}: OptionStackProps & WithTransition): React.ReactElement => {
+  const titleStack = optionPropsz.map((optionProps, index, array) => {
```

Comment:
`{useTitle}`

Reply:
Actually, no. It's useful to explain the z pluralisation of the `optionPropsz` before it.